packages/did-store-interface/src/interface.ts

```diff
@@ -0,0 +1,15 @@
+
+export interface IDidStore {
```

Comment:
Add method `delete(uri: string)` to the `did-store-interface`. 
For IPFS implementation, you can not delete the content as such but you can display an error "Not supported with IPFS."

Reply:
done